### PR TITLE
add addon filter for addonmanger

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

File: pkg/manager/controllers/addonowner/controller.go [hunk 1]
```diff
@@ -80,9 +86,10 @@ func (c *addonOwnerController) sync(ctx context.Context, syncCtx factory.SyncCon
 	}
 
 	if !c.addonFilterFunc(clusterManagementAddon) {
+		klog.Infof("\n ### cma not filtered: %v+", clusterManagementAddon)
 		return nil
 	}
-
+	klog.Infof("\n ### cma filtered: %v+", clusterManagementAddon)
```
Comment (qiujian16) [inline pkg/manager/controllers/addonowner/controller.go]: this line is unnecessary

Comment (zhiweiyin318) [reply pkg/manager/controllers/addonowner/controller.go]: removed

**Pull request merged**